### PR TITLE
[5.5] Fixed cache repository getMultiple

### PR DESCRIPTION
**Pull request by GrahamCampbell**
TLDR: Some implementations use a `stdClass` as the `$default` param, and we are expected to support both in order to fulfil the PSR specification `Psr\SimpleCache\CacheInterface` we implement.

---

In particular, without this fix, I am getting the following error:

```
Error Cannot use object of type stdClass as array 
    .../vendor/laravel/framework/src/Illuminate/Cache/Repository.php:131 Illuminate\Cache\Repository::getMultiple
    .../vendor/symfony/cache/Adapter/SimpleCacheAdapter.php:45 Symfony\Component\Cache\Adapter\SimpleCacheAdapter::doFetch
    .../vendor/symfony/cache/Adapter/AbstractAdapter.php:164 Symfony\Component\Cache\Adapter\AbstractAdapter::getItem
    .../vendor/php-http/cache-plugin/src/CachePlugin.php:143 Http\Client\Common\Plugin\CachePlugin::doHandleRequest
```

* Line 131 of `Repository.php` (`v5.5.40` of laravel/framework) is the line that tries to do an `isset` test on `$default` which won't work unless we have an array (or a class that is array accessable).
* Line 45 of `SimpleCacheAdapter.php` (`v3.4.29`) is the line that calls `getMultiple` with a `stdClass` as the `$default` parameter.

With this fix in place, the `stdClass` is converted to an `array` before the `isset` is run on it. Moreover, when the code then goes to call `many` later, it is passing a genuine `array`, as is required.

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Actually, Laravel's implementation is totally wrong. This default is not meant to represent a map. It is meant to represent the single default value... I'll resubmit this soon.
